src/processor/CodeSystemProcessor.ts

```diff
@@ -18,6 +18,9 @@ export class CodeSystemProcessor {
 
   static extractRules(input: any, target: ExportableCodeSystem, fisher: utils.Fishable): void {
     const newRules: ExportableCodeSystem['rules'] = [];
+    input.concept?.forEach((concept: any) => {
+      newRules.push(new ExportableConceptRule(concept.code, concept.display, concept.definition));
+    });
     newRules.push(...CaretValueRuleExtractor.processResource(input, fisher, input.resourceType));
```

Comment:
We might want to reverse the order of these, so that `CaretValueRule`s are written first, before the concepts, since I think that is the convention we generally follow.